boost/meta.yaml

```diff
@@ -8,6 +8,7 @@ source:
   md5: 6aa9a5c6a4ca1016edd0ed1178e3cb87
 
 build:
+  binary_relocation: False      [linux]
```

Comment:
I'm sorry, I haven't read all the comments to the PR before asking. However, I still don't get how you apply `RPATH` to the binaries then. Does it magically get applied by your old gcc?

I've used boost package intensively in my packages and, in fact, I have added Python 3 support to this boost conda-package. I haven't noticed any problems with it on CentOS 6.7 and Ubuntu 14.04. If it is a bug in conda-build, then it should be fixed instead of applying this quite a weird workaround.